indico/web/flask/errors.py

```diff
@@ -38,22 +38,12 @@
 
 @errors_bp.app_errorhandler(NotFound)
```

Comment:
I think this whole handler can now go away as it is covered by `handle_http_exception`